src/storage/checkpoint/table_data_writer.cpp

```diff
@@ -137,7 +138,11 @@ void TableDataWriter::FlushSegment(idx_t col_idx) {
 		data_pointer.row_start = last_pointer.row_start + last_pointer.tuple_count;
 	}
 	data_pointer.tuple_count = tuple_count;
-	data_pointers[col_idx].push_back(data_pointer);
+	//! FIXME: Can't deal with strings yet
+	idx_t type_size = stats[col_idx]->type == TypeId::VARCHAR ? 0 : stats[col_idx]->type_size;
+    memcpy(&data_pointer.min_stats,stats[col_idx]->minimum.get(), type_size);
```

Comment:
Could you run the formatter (`python3 scripts/format.py master`)? It seems some code is not aligned properly.